Bootstrap/JS/index.js

```diff
@@ -0,0 +1,85 @@
+//username regex check if contains only letters and numbers and underscore
+var usernameRegex = /^\w{10,30}$/;
+// email regex check if contains dots and letters and ends with @vatrin.com
+var emailRegex = /^[A-Za-z]+[A-Za-z\.]+$/;
+//password regex check if have at least one lower case, one upper case, one digit and at least 8 length.
+var passwordRegex =
+  /(?=.*\d)(?=.*[a-z])(?=.*[A-Z])(?=.*[!@#\$%\^&\*\(\)_\+\-=\[\]{};':"\\\|,\.<>\/\?]).{8,}/;
+//mobile regex check if start with 05 and have 8 numbers after 05.
+var mobileRegex = /^\d{8}$/;
+
+const registerForm = document.querySelector("form#register-form");
+const loginForm = document.querySelector("form#login-form");
+
+if (registerForm) {
+  var usernameInput = document.getElementById("validationTooltipUsername");
+  var emailInput = document.getElementById("validationTooltip01");
+  var passwordInput = document.getElementById("validationTooltip02");
+  var confirmPasswordInput = document.getElementById("validationTooltip03");
+  var mobileInput = document.getElementById("validationTooltip04");
+
+  const usernameMessage = document.querySelector("#usernameErrorMessage");
+  const emailMessage = document.querySelector("#emailErrorMessage");
+  const passwordMessage = document.querySelector("#passwordErrorMessage");
+  const confirmPasswordMessage = document.querySelector(
+    "#confrimPasswordErrorMessage"
+  );
+  const mobileMessage = document.querySelector("#mobileErrorMessage");
```

Comment:
find a way to get rid of those variables

Reply:
can you direct me how to do so?